flake.nix

```diff
@@ -0,0 +1,18 @@
+{
+  inputs = {
+    nixpkgs.url="github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+  outputs = { self, nixpkgs, flake-utils }@inputs:
+  flake-utils.lib.eachDefaultSystem (system:
+  let pkgs = import nixpkgs { inherit system; };
+  in {
+    devShells.default = pkgs.mkShell {
+      packages = with pkgs; [
+        ansible
+        ansible-lint
+      ];
+      PASSWORD_STORE_DIR="~/.password-store/chaos/jetzt";
```

Comment:
I'm not sure whether to include a `flake.nix` in this repo. Especially L15 makes hard assumptions about the system of the persons using it. On my system e.g. the password store is not located in `~/.password-store`.